crates/nu-command/src/filters/update.rs

```diff
@@ -116,6 +116,10 @@ fn update(
     let engine_state = engine_state.clone();
     let ctrlc = engine_state.ctrlc.clone();
 
+    // Let's capture the metadata for ls_colors
+    let metadata = input.metadata();
+    let mdclone = metadata.clone();
```

Comment:
i see `metadata` is cloned below, why not use `mdclone` there or always use `.clone`? :yum:

Reply:
I couldn't figure out how to do it with just metadata because it's `move`d into the closure. If someone has a better idea, I'm all ears. This the error it gives with just trying to use metadata, even though I cloned it everywhere I used it.
```
❯ cargo r                                                                                                                                                         14:00:25
   Compiling nu-command v0.84.1 (C:\Users\dschroeder\source\repos\forks\nushell\crates\nu-command)
error[E0382]: borrow of moved value: `metadata`
   --> crates\nu-command\src\filters\update.rs:175:27
    |
120 |     let metadata = input.metadata();
    |         -------- move occurs because `metadata` has type `std::option::Option<Box<PipelineMetadata>>`, which does not implement the `Copy` trait
...
134 |                 move |mut input| {
    |                 ---------------- value moved into closure here
...
155 |                         input_at_path.into_pipeline_data_with_metadata(metadata.clone()),
    |                                                                        -------- variable moved due to use in closure
...
175 |             .set_metadata(metadata.clone()))
    |                           ^^^^^^^^^^^^^^^^ value borrowed here after move

For more information about this error, try `rustc --explain E0382`.
error: could not compile `nu-command` (lib) due to previous error
```

Reply:
oooh i see :thinking: 

yeah i'm not sure, maybe this is not critical, just wanted to make sure it was not something you planned to remove :wink: